common/app/views/fragments/containers/facia_cards/videoContainer.scala.html

```diff
@@ -0,0 +1,75 @@
+@(containerDefinition: layout.FaciaContainer, frontProperties: model.FrontProperties)(implicit requestHeader: RequestHeader)
+
+@import common.LinkTo
+@import model.{VideoPlayer, ImageOverride, FaciaDisplayElement}
+@import views.support.{ImgSrc, Video700}
+@import layout.{ColumnAndCards, ContentCard, FaciaCardAndIndex, FaciaCardHeader}
+@import views.html.fragments.items.facia_cards.item
+@import views.html.fragments.items.elements.facia_cards.{itemImage, title}
+@import views.html.fragments.media.{video, formattedDuration}
+
+<div class="fc-container__inner">
+    <h2 class="video-title fc-container__header__title">
+        <a href="@containerDefinition.href">@containerDefinition.displayName</a>
+    </h2>
+</div>
+
+<div class="video-playlist video-playlist--0 js-video-playlist"
+     data-number-of-videos="@(containerDefinition.collectionEssentials.items.zipWithIndex.length - 1)">
+    <div class="video-playlist__control video-playlist__control--prev js-video-playlist-prev">
+        @fragments.inlineSvg("chevron-left", "icon", Seq("video-playlist__icon"))
+    </div>
+    <div class="video-playlist__control video-playlist__control--next js-video-playlist-next">
+        @fragments.inlineSvg("chevron-right", "icon", Seq("video-playlist__icon"))
+    </div>
+
+    <ul class="u-unstyled video-playlist__inner">
+        <li class="video-playlist__item video-title video-title--leftcol fc-container__header__title">
+            <a class="video-title__link" href="@containerDefinition.href">@containerDefinition.displayName</a>
+        </li>
+
+        @containerDefinition.collectionEssentials.items.zipWithIndex.map { case (item, index) =>
+            @if(item.header.isVideo) {
+                <li class="video-playlist__item video-playlist__item--@index">
+                    @item.properties.maybeContent.map { content => 
+                        <div class="video-playlist__content">
+                            <div class="video-playlist__headline">
+                                @title(FaciaCardHeader.fromTrail(item, None), 0, 0, withLink = false)
+                                @if(item.properties.showByline) {
```

Comment:
qq unrelated to this PR, what/who sets this property? I want to make it mandatory, i.e always credit the video producer, in a similar way we do images.